lib/main_common.pm

```diff
@@ -484,8 +484,9 @@ sub load_system_role_tests {
     if (!get_var("REMOTE_CONTROLLER") && !check_var('BACKEND', 'ipmi') && !is_hyperv_in_gui && !get_var("LIVECD")) {
         loadtest "installation/logpackages";
     }
-    loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS');
+    loadtest "installation/disable_online_repos"       if get_var('DISABLE_ONLINE_REPOS');
     loadtest "installation/installer_desktopselection" if is_opensuse;
+    loadtest "installation/system_role"                if is_caasp('kubic');
```

Comment:
You don't need ('kubic') parameter, `if is_caasp` is enough. The same on all places - will caasp 4.0 share this installation workflow?

Reply:
I don't know, so I wrote defensively for now

Even if CaaSP 4 follows a similar installation workflow, it will be more like SLE 15 than Tumbleweed, which will likely mean running the system_role at a different point

That'll be a mess we can worry about when we get there :)